lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -81,12 +83,19 @@ class ScreenshotThumbnails extends Audit {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const cachedThumbnails = new Map();
 
-    return artifacts.requestSpeedline(trace).then(speedline => {
+    return Promise.all([
+      artifacts.requestSpeedline(trace),
+      TTFI.audit(artifacts).catch(() => ({})),
```

Comment:
not IMO, if it's erroring from the perspective of this audit we don't care at all since we don't need to adjust the scale, it will already be flagged and raised by TTFI if you cared about TTFI and if you didn't then this shouldn't either